debian/rules

```diff
@@ -149,7 +150,7 @@ override_dh_installinit:
 	dh_installinit -pcloudstack-management -pcloudstack-agent -pcloudstack-usage --onlyscripts --no-start
 
 override_dh_systemd_enable:
-	dh_systemd_enable -pcloudstack-agent -pcloudstack-usage
+	dh_systemd_enable -pcloudstack-management -pcloudstack-agent -pcloudstack-usage
```

Comment:
Shouldn't we somehow clean up any left behind script in /etc/init.d of the management server?